src/components/BeaconsForm.tsx

```diff
@@ -28,20 +28,13 @@ export const BeaconsForm: FunctionComponent<BeaconsFormProps> = ({
   formErrors = [],
   errorMessages = [],
   pageText = null,
-  includeUseIndex = true,
 }: BeaconsFormProps): JSX.Element => {
-  const backButton: ReactNode = includeUseIndex ? (
-    <BackButtonRouterIndexes href={previousPageUrl} />
-  ) : (
-    <BackButton href={previousPageUrl} />
-  );
-
   const pageTextComponent: ReactNode =
     typeof pageText === "string" ? <GovUKBody>{pageText}</GovUKBody> : pageText;
 
   return (
     <Layout
-      navigation={backButton}
```

Comment:
`BackButtonRouterIndexes` was handling business logic such as "If user hasn't specified a BeaconUse, redirect to page X" a few levels deep in the React component tree.  Replaced with a `Rule`.